src/mainboard/ast/ast25x0/Makefile.toml

```diff
@@ -27,10 +28,5 @@ args = ["xbuild", "@@split(CARGO_ARGS, )"]
 
 [tasks.run]
 dependencies = ["default"]
-command = "qemu-system-arm"
-args = ["-machine", "virt", "-bios", "${TARGET_DIR}/oreboot.bin", "-nographic", "-S", "-s", ]
-
-[tasks.2500]
-dependencies = ["default"]
-command = "qemu-system-arm"
-args = ["-m", "512m", "-machine", "ast2500-evb", "-bios", "${TARGET_DIR}/oreboot.bin", "-nographic", ]
+command = "/home/ryanoleary/repos/openbmc/qemu/build-arm/arm-softmmu/qemu-system-arm"
```

Comment:
this one won't end well, I'm not you :-)

Reply:
Whooops. I guess the important thing is to use openbmc qemu -- they've emulated more of the soc.

Reply:
Done!